pandas/tests/series/test_timeseries.py

```diff
@@ -1,26 +1,10 @@
 import numpy as np
 
-import pandas as pd
 from pandas import DataFrame, Series, date_range, timedelta_range
 import pandas._testing as tm
 
 
 class TestTimeSeries:
-    def test_contiguous_boolean_preserve_freq(self):
```

Comment:
where did this go?

Reply:
test_arithmetic_with_duplicate_index

Reply:
woops, meant test_getitem_boolean_contiguous_preserve_freq